runtime/testing/e2e/a11y.ts

```diff
@@ -0,0 +1,46 @@
+/*jslint node: true */
+'use strict';
+
+const AxeBuilder = require('axe-webdriverjs');
+import { browser } from 'protractor';
+
+export class A11yTest {
+
+  public static log(results): any {
+      //console.error(`${violations}`);
+      var numResults = results.violations.length;
+
+      if (numResults > 0) {
+        console.error("Accessibility failure(s) found for: " + results.url + "\n");
+        results.violations.forEach(result => {
+          var label = result.nodes.length === 1 ? 'element' : 'elements';
+          var msg = result.nodes.reduce(function(msg, node) {
+              return `${msg}  Location: ${node.target[0]}
```

Comment:
this is intentional so each failure has a header line and then each failure instance found is indented under it.. 

```
1 element failed 'label' rule: Form elements must have labels 
  Location: body > sky-pages-app > ng-component > my-home > input
  <input id="state" name="state" type="text">
  Get help at: https://dequeuniversity.com/rules/axe/2.2/label?application=webdriverjs
1 element failed '"next rule"....
```